### PR TITLE
Add post about ABRT client ansible FAF url setup

### PR DESCRIPTION
**Pull request by juliusmilan**

### CODE REVIEW

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "FAF url setup of ABRT client Ansible role"
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: URL

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+date:   2017-10-15 10:31:14 +0100
+author: Julius Milan <jmilan@redhat.com>
+visible: 1
+categories: abrt ansible faf url setup
+---
+We recently added a new option to ABRT client [Ansible role][ansible-link] to setup
+url of the FAF server, i.e. where crash reports from ABRT client are reported.
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: URL

Comment (xsuchy) [reply _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: server, i.e., where

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+This small improvement will mostly appreciate people using or willing to use their own
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: It appears that own is a tautology. Just remove it.

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+installation of FAF for gathering crash reports from ABRT client for custom analysis.
+While their own FAF installation is running on custom server (easy to do using
+[FAF ansible role][faf-ansible-link]) or have their FAF running [in container][faf-in-container-link].
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: "While" makes this sentence a fragment. It should be while-then, while-do, while ... the others ...

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+# Usage
+To use ABRT client Ansible role, it is needed to simply declare it in your playbook, as:
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: To use ABRT client Ansible role it is needed to declare it ...
(comma, and simply)

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+```
+   ...
+   roles:
+     - ansible-abrt-client-role
+   ...
+```
+By default, FAF url is set to ```https://retrace.fedoraproject.org/faf```, which is main FAF
+installation of Red Hat.
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: of Fedora.

File: _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+   roles:
+     - { role: ansible-abrt-client-role, faf_url: 'your.faf.url' }
+   ...
+```
+Or, using the newer syntax:
+```
+   ...
+   tasks:
+   - include_role:
+        name: ansible-abrt-client-role
```
Comment (xsuchy) [inline _posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown]: there is one extra space. It should be aligned with faf_url bellow.

**Comment by juliusmilan** [comment]
Thanks for review, updated, all review requirements fixed

**Pull request merged**